test/units/tests.ts

```diff
@@ -182,4 +182,31 @@ describe('VSOClient Units', function () {
         //Assert
         assert(res.id === "testLocation");
     });
+});
+
+describe('WebApi Units', function () {
+    const osName = os.platform(); 
+    const osVersion = os.release();
+
+    before(() => {
+    });
+
+    after(() => {
+    });
+
+    afterEach(() => {
+    });
+
+    it('sets the user agent correctly when request settings are specified', async () => {
+        const myWebApi: WebApi.WebApi = new WebApi.WebApi('microsoft.com', WebApi.getBasicHandler('user', 'password'),
+                                                          undefined, {productName: 'name', productVersion: 'version'});
+        const userAgent: string = 'name/version (vsts-node-api 6.6.1; ' + osName + ' ' + osVersion + ')';
```

Comment:
Should we load version dynamically for each of these tests? Otherwise unit tests will break every time we change package version.